messages/run-dfa.js

```diff
@@ -27,12 +17,10 @@ module.exports = {
 	"validations": {
 		"methodLevelTargetCannotBeGlob": "Method-level targets supplied to --target cannot be globs",
 		"methodLevelTargetMustBeRealFile": "Method-level target %s must be a real file",
-		"projectdirCannotBeGlob": "--projectdir cannot specify globs",
-		"projectdirMustBeDir": "--projectdir must specify directories",
-		"projectdirMustExist": "--projectdir must specify existing paths"
+		"projectdirIsRequired": "--projectdir is required for this command",
```

Comment:
@teresa-allen-sfdc , this is a net-new message and requires approval. Context as follows:
To define `--projectdir` in one place and have it used by both commands, I needed to declare it as optional for both, which meant I needed to add a manual check for its presence in `scanner:run:dfa`. If that check fails, this error message is displayed.